decode.go

```diff
@@ -436,8 +436,7 @@ func parseTime(s string) (time.Time, error) {
 		modifier = 24 * time.Hour
 	}
 
-	norwegianLocation, err := time.LoadLocation("Europe/Oslo")
-	t, err := time.ParseInLocation(gs2TimeLayout, s, norwegianLocation)
+	t, err := time.Parse(gs2TimeLayout, s)
```

Comment:
```suggestion
	t, err := time.ParseInLocation(gs2TimeLayout, s, time.UTC)
```
slik at det alltid blir lest som utc uansett hvor vi kjører.